Configuration/Settings.yaml

```diff
@@ -7,6 +7,10 @@ TYPO3:
     presets:
       default:
         title: 'Default'
+        renderingOptions:
+          # Package with XLIFF files for translation of form labels, placeholders, ...
+          # If you need custom messages, copy Resources/Private/Translations/en/Main.xlf to your package and adjust this setting
+          translationPackage: 'TYPO3.Form'
```

Comment:
This setting is not expected/evaluated here. And the now missing `formElementTypes.'TYPO3.Form:Base'.renderingOptions.translationPackage` setting leads to an exception ```#1416832309: The current package key can't be resolved.```.
So instead, we should keep this setting (line 31) and *add* the `validationErrorTranslationPackage` setting on the same level